agent/assets/installconfig/install-config_yaml.j2

```diff
@@ -70,18 +70,9 @@ pullSecret: {{ pull_secret_contents }}
 sshKey: {{ ssh_pub_key }} 
 {% if mirror_images == "true" %}
 imageContentSources:
+{{ image_content_sources }}
 {% if mirror_command == "oc-mirror" %}
```

Comment:
The two `if`s can be combined into one. Otherwise, LGTM.

Reply:
Fixed